packages/components/Icon/CHANGELOG.json

```diff
@@ -1,6 +1,27 @@
 {
   "name": "@fluentui-react-native/icon",
   "entries": [
+    {
+      "date": "Thu, 05 Jan 2023 21:10:19 GMT",
+      "tag": "@fluentui-react-native/icon_v0.17.0",
+      "version": "0.17.0",
+      "comments": {
+        "undefined": [
+          {
+            "author": "beachball",
+            "package": "@fluentui-react-native/icon",
+            "comment": "Bump @fluentui-react-native/framework to v0.8.30",
+            "commit": "681dfc9e0dbbd686f75c57f56c0aad9bdf346eb1"
+          },
+          {
+            "author": "beachball",
+            "package": "@fluentui-react-native/icon",
+            "comment": "Bump @fluentui-react-native/text to v0.19.8",
+            "commit": "681dfc9e0dbbd686f75c57f56c0aad9bdf346eb1"
+          }
+        ]
+      }
+    },
```

Comment:
the command inserted this into the CHANGELOG.json... not sure if it should be kept or not. I think this is what was causing trouble last time because @fluentui-react-native/icon is already on 0.17.0